modules/course_management/cog.py

```diff
@@ -0,0 +1,56 @@
+from utils.utils import build_aliases
+from utils.embedder import embed_success
+from discord.ext.commands import has_permissions
+from discord.ext import commands
+from modules.course_management.course_adder import CourseAdder
+import config
+
+
+class CourseManagerCog(commands.Cog):
+	def __init__(self):
+		self.adder = CourseAdder(config.conn, config.sql_fetcher)
+
+	@has_permissions(manage_channels=True)
+	@commands.command(
+		**build_aliases(
+			name="courses.add",
+			prefix=("course", "courses"),
+			suffix=("add", "new"),
+			more_aliases=("addcourse",),
+		)
+	)
+	async def add_course(
```

Comment:
Consistency since has_permissions() usually goes after command() in the docs

```suggestion
	@commands.command(
		**build_aliases(
			name="courses.add",
			prefix=("course", "courses"),
			suffix=("add", "new"),
			more_aliases=("addcourse",),
		)
	)
	@has_permissions(manage_channels=True)
	async def add_course(
```